gatsby_fp/src/components/Layout/Body/Fields/ProfSocieties.js

```diff
@@ -2,32 +2,32 @@ import React from "react"
 const shortid = require("shortid")
 
 export default function ProfSocieties({ profSocietiesList }) {
+
+  const liInnerHTMLList =
+    profSocietiesList.map(element => {
+      let liString = '';
+      if (element.orgabbr) {
+        liString += `${element.orgabbr} `
```

Comment:
No punctuation separating any of these? The corresponding negative diff has a dash after this.

Reply:
It may actually be appropriate for the dash to be its very own if statement, leaving the existing if statements exactly as they are. The new if statement would be to put the dash only in the case of `orgabbr` and `org` both existing.